org-code-javabuilder/theater/src/main/java/org/code/Image.java

```diff
@@ -0,0 +1,67 @@
+package org.code;
+
+public class Image {
+    /**
+     * Creates a new image object, using the pixel information from the file
+     * uploaded to the asset manager.
+     * 
+     * @param filename the name of the image loaded into the asset manager for the
+     *                 project
+     */
+    public Image(String filename) throws ImageException;
```

Comment:
If we're creating custom exceptions that will be visible to the student, we should also add them to the api so they are documented.

Reply:
Also, in what cases would the ImageException be thrown?

Reply:
Oh, I may have missed this - it should be `FileNotFoundException`